docs/discovery.md

```diff
@@ -116,11 +116,11 @@ You can list the nodes in your cluster.
 On each of your nodes, start the Swarm agent. The node IP address
 doesn't need to be public as long as the Swarm manager can access it.
 
-    swarm join --advertise=<node_ip:2375> consul://<consul_addr>/<optional path prefix>
+    swarm join --advertise=<node_ip:2375> consul://<consul_addr1>,<consul_addr2>/<optional path prefix>
```

Comment:
When done, it should be done consistently, I agree. Perhaps you could create a separate issue (so that we don't delay merging this PR). It was just something I noticed :smile: